modules/home-manager/wezterm.nix

```
@@ -0,0 +1,29 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  cfg = config.catppuccin.wezterm;
+in
+
+{
+  options.catppuccin.wezterm = catppuccinLib.mkCatppuccinOption { name = "wezterm"; };
+
+  imports = catppuccinLib.mkRenamedCatppuccinOptions {
+    from = [
+      "programs"
+      "wezterm"
+      "catppuccin"
+    ];
+    to = "wezterm";
+  };
+
+  config = lib.mkIf cfg.enable {
+    programs.wezterm = {
+      extraConfig = ''
+        return {
+          color_scheme = "Catppuccin ${catppuccinLib.mkUpper cfg.flavor}",
+        }
+      '';
```

although this is nice, this would break alot of configs beacuse we couldn't set other options. Instead we should use something like this, though that would enforce users to fit into their configuration.
```suggestion
      extraConfig = lib.mkBefore ''
        local config = {}
        if wezterm.c_builder then
          config = wezterm.config_builder()
        end
       
        require("${sources.wezterm}/init.lua").apply_to_config(config)
      '';
```

---

Yeah, I tried to add more config after PR creation and it all broke. I believe your solution is the best until home-manager module will be expanded with proper configuration options.